src/commands/dev/mod.rs

```diff
@@ -88,6 +88,17 @@ fn preview_request(
     let now: DateTime<Local> = Local::now();
     let preview_id = &preview_id;
 
+    let mut headers: HeaderMap = HeaderMap::new();
+
+    for header in &parts.headers {
+        let (name, value) = header;
+        let forward_header = format!("cf-ew-raw-{}", name);
+        // TODO: remove unwrap
+        let header_name = HeaderName::from_bytes(forward_header.as_bytes()).unwrap();
```

Comment:
This function returns a Result with error type failure::Error, can we just get rid of this unwrap now? If the error type returned by calling `?` does not work with failure::Error, we can just capture the error and wrap it in a failure::Error

Reply:
This function actually returns a ResponseFuture, and I'm not quite sure how to make that play nicely with the failure crate which is why i have the TODO statements. cc @ashleygwilliams